scripts/install-deps.sh

```diff
@@ -527,7 +527,7 @@ do_install_azure_storage_sdk() {
 
     cmake "${azure_storage_sdk_cmake_options[@]}" . || return
 
-    cmake --build . || return
+    cmake --build . || return 1
```

Comment:
nit: Change to common exit pattern unless there's reasons to have it return 1

Reply:
Fixed